packages/d/distrobox/package.yml

```diff
@@ -0,0 +1,16 @@
+name       : distrobox
+version    : 1.7.2.1
+release    : 1
+source     :
+    - https://github.com/89luca89/distrobox/archive/refs/tags/1.7.2.1.tar.gz : ff2cca0c6334fff6ed577d23f68a6746ad4009f42d8a45eef5ca3850c895a4bb
+homepage   : https://distrobox.it/
+license    : GPL-3.0
```

Comment:
Use a valid [SPDX](https://spdx.org/licenses/) license identifier. It is either `GPL-3.0-only` or `GPL-3.0-or-later`. Please look into it, which one distrobox uses.

Reply:
Looking at the repo it just states GPL-3.0. I'll take a look at the Fedora package to see what they have it as.

Reply:
If it is not stated clearly, it usually mean `GPL-3.0-or-later`, but please look what other distro do.

Reply:
Fedora & Gentoo have it labeled as GPLv3, Alpine, Arch, Homebrew, openSuse have it as GPL-3.0-only, Debian/Ubuntu has it as GPL-3.0-later. So GPL-3.0-only?

Reply:
Let's use the majority, `GPL-3.0-only`